code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2595,7 +2595,7 @@
 		var/datum/organ/external/chest/C = H.get_organ(LIMB_CHEST)
 		for(var/datum/organ/internal/I in C.internal_organs)
 			if(I.damage > 0)
-				I.damage -= 0.20
+				I.damage = max(0,I.damage-=0.20)
```

Comment:
should just be `max(0, I.damage - 0.20)`, no need to use the assignment operator in there

Reply:
thanks for catching that. Coding at 3AM with a christmas headache is not recommendable to anyone.